src/main/preload.d.ts

```diff
@@ -112,7 +118,7 @@ declare interface Window {
     getBackups: typeof import('../main/device/service').getBackupsFromDevice;
 
     addBackup: typeof import('../main/device/service').addBackup;
-    addBackupsFromLocalPaths: typeof import('../main/device/service').createBackupsFromLocalPaths;
```

Comment:
This is needed for the onboarding, why is being removed here? It will make the onboarding crash

https://github.com/internxt/drive-desktop/blob/4ef2cd07563f917a9c80dadf2e3c032fe6ffa974/src/renderer/pages/Onboarding/index.tsx#L27

Reply:
Probably this was because of the linter. I do not remember deleting those lines, either is necessary for the task. Will undo them

Reply:
Don't know why it's marked as deleted. It is not for my local